scripts/qemu/ia32-generic.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Shell script for running Phoenix-RTOS on QEMU (ia32-generic)
+#
+# Copyright 2021 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+CMD="qemu-system-i386"
+
+if [ "$#" -eq 0 ]; then
+	# No args (default command)
+	CMD+=" -hda ../../_boot/phoenix-ia32-generic.disk"
+else
+	hd=0
+	vblk=0
+	# Parse args
+	while [ "$#" -gt 0 ]; do
+		case "$1" in
+			# Load kernel (.elf)
+			-kernel)
+				CMD+=" -kernel $2"
+				shift
+				shift
```

Comment:
```suggestion
				shift 2

```
shorter

Reply:
Added in next commit, thanks.

Reply:
@pawelpisarczyk Done. I've added separate script for each device we want to emulate with QEMU.